api/alert_channels.go

```diff
@@ -89,6 +89,7 @@ const (
 	VictorOpsAlertChannelType
 	CiscoSparkWebhookAlertChannelType
 	MicrosoftTeamsAlertChannelType
+	SplunkAlertChannelType
```

Comment:
Let us follow a standard of `[Name]AlertChannelType`
```suggestion
	SplunkHecAlertChannelType
```